daemon/algod/api/algod.oas2.json

```diff
@@ -1600,10 +1784,6 @@
           "type": "integer",
           "x-go-name": "AssetID"
         },
-        "creator": {
-          "description": "Address that created this asset. This is the address where the parameters for this asset can be found, and also the address where unwanted asset units can be sent in the worst case.",
```

Comment:
Will this impact sdk backwards compatibility? Please comment.

Reply:
As is, yes it will. I'd like to suggest we manually retain this field in the SDK models so they don't have an explicitly breaking change. The field can just be empty if algod doesn't return it.

Reply:
IMO it would be preferable to introduce a breaking change than provide an empty value. That way you would find out  that you need to pull the creator from /v2/assets/{asset-id} instead. However I don't have a lot of experience with how we deprecate functionality in the SDKs or warn users of issues like this, whether a compiler error or compiler warning is more typical.

Reply:
Related: https://github.com/algorand/indexer/pull/900#discussion_r815128273